src/LineageTree/lineageTree.py

```diff
@@ -129,7 +129,7 @@ def add_branch(
                     )
                     pred = _next
         else:
-            for t in range(length):
+            for t in range(1, length):
```

Comment:
where that 1 comes from?

Reply:
You are correct there was a problem with the time the node was being added. It is fixed now.